tests/utils.py

```diff
@@ -38,7 +38,12 @@ def raises_unsupported_algorithm(reason):
     assert exc_info.value._reason is reason
 
 
-def load_vectors_from_file(filename, loader, mode="r"):
+T = typing.TypeVar("T")
+
+
+def load_vectors_from_file(
+    filename, loader: typing.Callable[..., T], mode="r"
```

Comment:
Going to merge this for now, but this shouldn't be a `...`, it should be an IO, right?

Reply:
Probably, I basically just skipped typing everything other than the return because that’s 99% of the value of the type here. We can type the rest in the future